splunk_add_on_ucc_framework/__init__.py

```diff
@@ -365,9 +365,17 @@ def remove_files(path):
         path (str): Path to remove *.egg-info and *.dist-info files.
     """
 
-    rmdirs = glob.glob(os.path.join(path, "*.egg-info")) + glob.glob(
-        os.path.join(path, "*.dist-info")
-    )
+    delete_list = ["*.egg-info", "*.dist-info"]
+    # ADDON-23349: Do not remove jsonschema dist-info/egg-info directory
+    do_not_delete_list = ["jsonschema-*"]
```

Comment:
I have seen other libs also require the dist-info. For example https://github.com/googleapis/google-api-python-client/blob/fd2965cda2776ef2f9e02fb720fd57f55948ede0/googleapiclient/model.py#L34.

Should we just not delete dist-info? WDYT @arys-splunk @rfaircloth-splunk?

Reply:
Looks like they both are still in use: https://packaging.python.org/discussions/wheel-vs-egg/
One is official, another is not, but still used.